.githooks/post-checkout

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Please remove these githooks.

Reply:
resolved in commit [dc89e4d](https://github.com/moreh-dev/MIOpen/pull/15/commits/dc89e4df1bb98749eeac4e445c9723145c05089b)